MAGN-8838 After copy/paste undo button is not refreshed

marimano opened the pull request:
### Purpose

- Undo/redo buttons are refreshed after copy-pasting;
- Repeating code is moved into a separate method

### Reviewers

@mjkkirschner

mjkkirschner commented:
@marimano is this to fix a specific bug? - was this bug introduced by your other copy/paste Prs?

marimano commented:
@mjkkirschner, Currently I have ctrl+drag PR opened, I fixed this issue [there](https://github.com/DynamoDS/Dynamo/pull/5491/files#diff-a2c9f949b12184fd558acbaa8323aba5R741).
I assume this PR will be merged earlier than that, so I will update ctrl+drag in order to use [RaiseCanExecuteUndoRedo](https://github.com/DynamoDS/Dynamo/pull/5507/files#diff-dbb33ecd12dabd7b9d04cf73e0685f03R1748), added here

mjkkirschner commented on src/DynamoCoreWpf/Views/Core/AnnotationView.xaml.cs:
```diff
@@ -183,23 +183,20 @@ private void GroupTextBlock_SizeChanged(object sender, SizeChangedEventArgs e)
 
-                ViewModel.WorkspaceViewModel.DynamoViewModel.UndoCommand.RaiseCanExecuteChanged();
-                ViewModel.WorkspaceViewModel.DynamoViewModel.RedoCommand.RaiseCanExecuteChanged();
+            //Record the value here, this is useful when title is poped from stack during undo
```
typo

mjkkirschner replied:
and could you clarify this comment a bit more... what value and where are you recording it?

mjkkirschner commented:
@marimano thanks for simplifying this and fixing this bug, just one small thing with a comment and then I'll merge it.

marimano commented:
@mjkkirschner, I didn't go into what [GroupTextBox_OnIsVisibleChanged method](https://github.com/DynamoDS/Dynamo/pull/5507/files#diff-bcb2d54d8ef97c787e1a33a983c943ceR183) is doing. I just replaced two calling RaiseCanExecuteChanged() with calling corresponding single method and inverted if statement:
```
if (condition) 
{ 
code blocks 
}
```
=>
```
if (!condition) return;

code blocks
```
So I am not sure I can extend that comment properly - I have only fixed typo

mjkkirschner commented:
@marimano theres no failures on the CI - merging this in.
